src/app/core/config/default-configuration.ts

```diff
@@ -127,6 +127,7 @@ export const nonCustomizableProperties: Readonly<NonCustomizableConfig> = {
     },
   },
   keyboardLayoutForVms: 'us',
+  sidebarWidth: '330',
```

Comment:
Why is this a string? Config allows numeric values.

Reply:
Because I use this value only for default tag value and tag value should be a string

Reply:
It's not clear. You can potentially use this value in many places. If it must be a string, explain it in a comment or in the name of the variable (for example: sidebarWidthDefaultTagValue), although I would prefer it to be a number.